pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
I think you can eliminate this file. So when run tests in automation it will catch all test cases.

Reply:
Without this file, the tests inside the `actions` folder are not able to use the fixtures in `conftest.py`. 

This file is not preventing the other files to be executed in automation.

![image](https://github.com/user-attachments/assets/86ec1621-b979-4243-ba26-0aef36d3afc4)

Reply:
Actually, I would suggest to include this file in the template repo